code/__DEFINES/vendors.dm

```diff
@@ -21,10 +21,11 @@
 #define MARINE_CAN_BUY_COMBAT_ARMOR (1<<19)
 #define MARINE_CAN_BUY_COMBAT_R_POUCH (1<<20)
 #define MARINE_CAN_BUY_COMBAT_L_POUCH (1<<21)
+#define MARINE_CAN_BUY_ETOOL (1<<22)
 
-#define MARINE_CAN_BUY_KIT (1<<22)
+#define MARINE_CAN_BUY_KIT (1<<23)
 
-#define MARINE_CAN_BUY_ALL ((1<<23) - 1)
+#define MARINE_CAN_BUY_ALL ((1<<24) - 1)
```

Comment:
Ermmmmmmmmmmmmmmm I don't know if this is a good idea, it might work because the compiler resolves this at compile time, but you can't actually use 1<<24 in bitflags due to BYOND runtime only storing signed 24-bit ints before moving on to floats

Reply:
![image](https://github.com/cmss13-devs/cmss13/assets/112824777/b6b86ac6-407c-4aa4-88b8-b27a9d85e2fd)
Creating a 25 bitflag breaks the vendors, but it is not broken with the current iteration ((24) -1)